package.json

```diff
@@ -31,8 +31,10 @@
   },
   "scripts": {
     "clean": "rm -rf test-artifacts",
-    "lint": "eslint .",
-    "test": "yarn clean && jest"
+    "lint": "yarn preinstall && eslint .",
+    "test": "yarn clean && jest",
+    "flow": "yarn preinstall && flow check",
```

Comment:
It looks like we weren't running `flow check` in our verification script and were missing similar errors being raised by our linter:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ templates/plugin/content/src/types.js:3:28

Cannot resolve module fusion-core.

     1│ // @flow
     2│
     3│ import type {Context} from 'fusion-core';
     4│
     5│ export type PluginServiceType = {
     6│   from: (

// and 12 other similar missing module errors
```